<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeWindow</class>
 <widget class="QMainWindow" name="RecipeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="ingredientScrollPane">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>69</y>
      <width>191</width>
      <height>491</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>189</width>
       <height>489</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>Ingredients</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QScrollArea" name="recipeScrollArea">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>29</y>
      <width>551</width>
      <height>481</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>549</width>
       <height>479</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitRecipeButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>520</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 9pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newRecipeButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>520</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 9pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>New recipe</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
